### PR TITLE
Broker-Trigger general pass

### PR DESCRIPTION
**Pull request by nachocano**
## Proposed Changes

- Updating comments general pass.
- Remove unnecessary parameters from functions.
- Updating formatting of some files.

**Release Note**

```release-note
NONE
```

### CODE REVIEW

**Comment by nachocano** [comment]
@Harwayne  ready to review

File: pkg/reconciler/v1alpha1/broker/broker.go [hunk 1]
```diff
@@ -217,7 +218,7 @@ func (r *reconciler) reconcile(ctx context.Context, b *v1alpha1.Broker) (reconci
 
 	svc, err := r.reconcileIngressService(ctx, b)
 	if err != nil {
-		logging.FromContext(ctx).Error("Problem reconciling ingress Service", zap.Error(err))
+		logging.FromContext(ctx).Error("Problem reconciling ingress service", zap.Error(err))
```
Comment (Harwayne) [inline pkg/reconciler/v1alpha1/broker/broker.go]: I would probably leave Service capitalized as it represents a K8s Service.

**Review by Harwayne — COMMENTED** [review]
Looks good, but there is a conflict. Fix it and we'll merge this.

File: pkg/reconciler/v1alpha1/broker/broker.go (hunk 1 above)
Comment (nachocano) [reply pkg/reconciler/v1alpha1/broker/broker.go]: Will do!

**Comment by nachocano** [comment]
@Harwayne ready to merge

**Pull request merged**